src/cortxfs/cortxfs_ops.c

```diff
@@ -716,44 +729,38 @@ int cfs_rename(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 			 * will be closed.
 			 */
 			log_trace("Detaching a file from the tree "
-				  "(%llu, %llu, %s)", *dino_dir, dino, dname);
-			RC_WRAP_LABEL(rc, out, cfs_detach, cfs_fs, cred,
-				      dino_dir, &dino, dname);
+				  "(%llu, %llu, %s)", *dino_dir, *dst_ino,
+				  dname);
+			RC_WRAP_LABEL(rc, out, cfs_detach2, ddir_fh, dst_fh,
+				      cred, dname);
 		}
-	} else {
-		ino_to_node_id(dino_dir, &dnode_id);
 	}
```

Comment:
Suggestion: For long if..else constructs, we can probably add a comment after closing brace for better readability. For example
```
if (overwrite_dest) {
..
..
} /* if (overwrite_dest) */
```

Reply:
Done.